docs/jwst/jump/description.rst

```diff
@@ -35,23 +35,39 @@ The two-point difference method is applied to each integration as follows:
 
 * Compute the first differences for each pixel (the difference between
   adjacent groups)
-* Compute the clipped median of the first differences for each pixel
+* Compute the clipped (dropping the largest difference) median of the first differences for each pixel.
 * Use the median to estimate the Poisson noise for each group and combine it
   with the read noise to arrive at an estimate of the total expected noise for
-  each group
+  each difference.
 * Compute the "difference ratio" as the difference between the first differences
-  of each group and the median, divided by the expected noise
+  of each group and the median, divided by the expected noise.
 * If the largest "difference ratio" is greater than the rejection threshold,
-  flag the group corresponding to that ratio as having a jump
+  flag the group corresponding to that ratio as having a jump.
 * If a jump is found in a given pixel, iterate the above steps with the
   jump-impacted group excluded, looking for additional lower-level jumps
-  that still exceed the rejection threshold
+  that still exceed the rejection threshold.
 * Stop iterating on a given pixel when no new jumps are found
 
 Note that any ramp values flagged as SATURATED in the input GROUPDQ array
 are not used in any of the above calculations and hence will never be
 marked as containing a jump.
 
+Multiprocessing
+===============
+This step has the option of running in multiprocessing mode. In that mode it will
+split the input data cube into a number of slices based on the number of available
+cores on the host computer and the value of the --max_cores input parameter. By
```

Comment:
In "--max_cores", the "--" might be confusing as it pertains to the command-line usage. Perhaps just "max_cores".

Reply:
I removed the "--" in front of max_cores.